account_invoice_edifact/models/account_move.py

```diff
@@ -40,8 +40,15 @@ def _edifact_invoice_get_interchange(self):
             sender_edifact, recipient_edifact, self.id, syntax_identifier
         )
 
+    def _edifact_invoice_get_address(self, partner):
+        if hasattr(partner, "street3"):
```

Comment:
```suggestion
        # oca/partner-contact/partner_address_street3 is installed
        if hasattr(partner, "street3"):
```

Reply:
please also add a reference to `https://github.com/OCA/edi/pull/904`
"we apply the same logic as"

Reply:
Thanks @nilshamerlinck, i updated it.